blocks/identity-block/components/bot-challenge-protection/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "@arc-fusion/prop-types";
+import { useFusionContext } from "fusion:context";
+import getProperties from "fusion:properties";
+import getTranslatedPhrases from "fusion:intl";
+import { Paragraph, useIdentity } from "@wpmedia/arc-themes-components";
+import useSales from "../../utils/useSales";
+import ReCAPTCHA from "react-google-recaptcha";
+
+const BotChallengeProtection = ({ challengeIn, setCaptchaToken, className }) => {
+	const { Identity, isInitialized } = useIdentity();
+  const { Sales } = useSales();
+	const [siteKey, setSiteKey] = useState();
+	console.log('siteKey', siteKey)
```

Comment:
remove consoleLog

Reply:
Looks like some attributes are not being used: error, setError, isAdmin, phrases, ...